### DISCUSSION TITLE
[backend] spring+webflux.fn+mongodb+spock

### DISCUSSION THREAD

**Discussion by alphawyu** [realworld-apps/realworld#932] [WIP Implementations]
- spring webflux functional router is "web on reactive stack". 
- spring data support reactive stream of NoSQL, and mongo db in this example. other NoSQL db includes cassandra, radis, (google cloud) firestore &c to name few.
- testing framework spock + spring webflux WebTestClient is easy to use, will be a addition to implement as a working example   

[https://github.com/alphawyu/spring-webflux.fn-mongodb-spock](https://github.com/alphawyu/spring-webflux.fn-mongodb-spock)

**Comment by geromegrignon** [comment]
Hi, thanks for working on this new implementation!

**Reply by alphawyu** [reply]
It is released this weekend. 
Have fun with it and help with your feedbacks. 
Enjoy!

**Comment by alphawy45** [comment]
will soon update to java 25, spring boot 3.5

**Reply by alphawy45** [reply]
published the update for java 25/spring boot 3.5
